server/tests/backends/fixtures/filter/dates_from_until_pypika.json

```diff
@@ -20,7 +20,7 @@
             {
               "column": "brewing_date",
               "operator": "until",
-              "value": "2022-01-07 00:00:00"
+              "value": "2022-01-07 01:00:00"
```

Comment:
Why did the input have to be changed ? We should keep the same contract

Reply:
https://github.com/ToucanToco/weaverbird/pull/1418/commits/8d6d27c067adf502a2ea3f6bd619470e99c8f58f